alerting/src/main/kotlin/org/opensearch/alerting/settings/LegacyOpenDistroAlertingSettings.kt

```diff
@@ -86,13 +86,26 @@ class LegacyOpenDistroAlertingSettings {
             Setting.Property.NodeScope, Setting.Property.Dynamic, Setting.Property.Deprecated
         )
 
+        val ALERT_FINDING_INDEX_MAX_AGE = Setting.positiveTimeSetting(
+            "opendistro.alerting.alert_finding_max_age",
+            TimeValue(30, TimeUnit.DAYS),
+            Setting.Property.NodeScope, Setting.Property.Dynamic, Setting.Property.Deprecated
+        )
+
         val ALERT_HISTORY_MAX_DOCS = Setting.longSetting(
             "opendistro.alerting.alert_history_max_docs",
             1000L,
             0L,
             Setting.Property.NodeScope, Setting.Property.Dynamic, Setting.Property.Deprecated
         )
 
+        val ALERT_FINDING_MAX_DOCS = Setting.longSetting(
+            "opendistro.alerting.alert_finding_max_docs",
+            1000L,
+            0L,
+            Setting.Property.NodeScope, Setting.Property.Dynamic, Setting.Property.Deprecated
+        )
+
```

Comment:
We do not want to make any new legacy settings. These settings are for backwards compatibility. We want everyone to use the new setting naming scheme. Put the default values here to the settings made in the `AlertingSettings` class.